### PR TITLE
TAJO-1597: Problem of ignoring theta join condition

### PR DESCRIPTION
**Pull request by jihoonson**
[Review on Reviewable](https://reviewable.io/reviews/apache/tajo/643)

### CODE REVIEW

**Comment by hyunsik** [comment]
The filters seem to be higher than sort. It may cause sorting of unnecessary rows.

**Comment by jihoonson** [comment]
Thanks! It was my mistake. I've fixed.

**Comment by hyunsik** [comment]
It seems to include in and exist subquery implementation.

File: tajo-plan/src/main/java/org/apache/tajo/plan/algebra/BaseAlgebraVisitor.java [hunk 1]
```diff
@@ -52,221 +52,221 @@ public RESULT visit(CONTEXT ctx, Stack<Expr> stack, Expr expr) throws TajoExcept
...
-    case TimestampLiteral:
-      current = visitTimestampLiteral(ctx, stack, (TimestampLiteral) expr);
-      break;
-    case IntervalLiteral:
-      current = visitIntervalLiteral(ctx, stack, (IntervalLiteral) expr);
-      break;
-
-    default:
-      throw new TajoInternalError("Cannot support this type algebra \"" + expr.getType() + "\"");
+      case SetSession:
```
Comment (hyunsik) [inline tajo-plan/src/main/java/org/apache/tajo/plan/algebra/BaseAlgebraVisitor.java]: The wrong coding format seems to be applied. {{switch}} and {{case}} should be the same column position.

**Comment by jihoonson** [comment]
Damn. It's my mistake.
I'll make another pr.

**Pull request closed without merging**